sdk/src/main/java/com/kin/ecosystem/base/KinEcosystemInitiator.java

```diff
@@ -16,7 +18,9 @@ private KinEcosystemInitiator() {
 	protected static void init(Context context) {
 		try {
 			Kin.initialize(context);
-		} catch (ClientException e) {
+			// If we had process restart we should load the account.
+			BlockchainSourceImpl.getInstance().createAccount();
```

Comment:
maybe worth changing the name to reflect it's either creating or loading the account?